lang/tags/keywords.talon

```diff
@@ -0,0 +1,3 @@
+tag: user.code_keywords
+-
+keyword (<user.code_keyword>+): user.code_keyword(code_keyword_list)
```

Comment:
Open to ideas on the prefix here.  

Fwiw I personally use "make" for keywords, statements, and operators, and then "snip" for the bigger structures where knausj tends to use "state", and they are actual vscode snippets

Reply:
I think we probably need to think about redesigning our programming command prefixes at some point. For now I think `keyword` is probably fine? It's very clear and specific.